scripts/jenkins/track-upstream-internal.sh

```diff
@@ -93,6 +93,7 @@ export OSC_BUILD_ARCH
 export JHOME=/home/jenkins
 export BS_CHECKOUT=$JHOME/${OBS_TYPE}_CHECKOUT/$OBS_PROJECT
 export OSC_BUILD_ROOT=$JHOME/buildroot
+export GIT_SSL_NO_VERIFY=true
```

Comment:
This not only disables verification for intranet hosts, but also for github.com .

Reply:
One solution while we wait for the new cert to be signed that avoids this problem is to add the self signed cert into the system trust store on the trackupstream node.

Reply:
Agreed. I used `git config --global http."https://git.suse.provo.cloud".sslVerify false` and `git config --global http."https://gerrit.prv.suse.net/".sslVerify false` instead on the trackupstream host. I updated #3387 to point this out.